hadoop-ozone/insight/src/main/java/org/apache/hadoop/ozone/insight/List.java

```diff
@@ -0,0 +1,38 @@
+package org.apache.hadoop.ozone.insight;
+
+import org.apache.hadoop.hdds.cli.HddsVersionProvider;
+import org.apache.hadoop.hdds.conf.OzoneConfiguration;
+
+import picocli.CommandLine;
+
+import java.util.Map;
+import java.util.concurrent.Callable;
+
+/**
+ * Subcommand to list of the available insight points.
+ */
+@CommandLine.Command(
+    name = "list",
+    description = "Show available insight points.",
+    mixinStandardHelpOptions = true,
+    versionProvider = HddsVersionProvider.class)
+public class List extends BaseInsightSubcommand implements Callable<Void> {
+
+  @CommandLine.Parameters(defaultValue = "")
+  private String selection;
```

Comment:
Do you plan to use this parameter, eg. to filter available insight point list?

Reply:
Yes, originally I used a tree like insight point structure and it helped to list only a subtree. When I switched to use a flat structure it was no longer supported.

I fixed it and restored this functionality. `ozone insight log scm` can list the scm related insight points from now...